server-spi/src/main/java/org/keycloak/models/AuthenticatedClientSessionModel.java

```diff
@@ -37,8 +37,16 @@
         public static final SearchableModelField<AuthenticatedClientSessionModel> TIMESTAMP  = new SearchableModelField<>("timestamp", Integer.class);
     }
 
+    String STARTED_AT_NOTE = "startedAt";
+
     String getId();
 
+    default int getStarted() {
```

Comment:
Perhaps `getStartedAt` ?

Reply:
Good point. I was thinking exactly about that and the reason why I did not is the fact UserSessionModel has method `getStarted()` . Not 100% sure whether to go with `getStartedAt`, which is probably slightly clearer name, or whether to stick with compatibility with UserSessionModel? Let me know your preference and I will do it that way.

Reply:
No preference now you gave your reasons. Up to you :)

Reply:
Ok :) I keep the "started" due the compatibility with UserSessionModel.